ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Util/LeakyBucket.hs

```diff
@@ -242,24 +347,24 @@ snapshot bucket = fst <$> snapshotFill bucket 0
 -- need to measure the time, and @io-classes@'s STM does not allow running IO in
 -- an STM.
 snapshotFill ::
-  (MonadSTM m, MonadMonotonicTime m) =>
+  ( MonadSTM m
```

Comment:
The comment of this function should now be mostly addressed, right?

Reply:
Indeed! Removed in 3f590a07ed5d4222147f1a2c4f3cf7fc6309aae8.